kiali/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Kiali Operator provided by Red Hat</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Using Kiali Operator provided by Red Hat</subtitle>
```

Comment:
In OCP docs, I usually see an article used with "Kiali Operator. For example:

```suggestion
<subtitle>Using the Kiali Operator provided by Red Hat</subtitle>
```

Reply:
done